aux_fcn.py

```diff
@@ -440,7 +440,9 @@ def prediction_parser(LFP,arch='CNN1D',model_number=1,new_model=None,n_channels=
         windowed_signal= model.predict(LFP,verbose=1)
         for i,window in enumerate(windowed_signal):
             y_predict[i*timesteps:(i+1)*timesteps]=window
-        
+    else:
+    	raise ValueError(f'The introduced architecture -{arch}- does not match the existing ones.')
```

Comment:
raises ValueErrorr if it does not enter into any of the elif of the architectures. It would be better to check from the beginning the input to rppleAI.predict() so it does not waste time normalizing and downsampling the LFP.